### PR TITLE
Upgrade to use 6.0.0 typings + remove all hard references to PIXI

### PR DESCRIPTION
**Pull request by SukantPal**
For some reason, the animation in the example isn't working right after this change. @ivanpopelyshev can you take a look.

### CODE REVIEW

**Comment by SukantPal** [comment]
The problem is that `import * as utils from '@pixi/utils`;` gets compiled into `PIXI` not `PIXI.utils`.

**Comment by SukantPal** [comment]
Fixed the problem. @pixi-build-tools/globals had @pixi/utils's namespace listed as PIXI, but it should've been PIXI.utils; republished.

File: package.json [hunk 1]
```diff
@@ -37,29 +37,32 @@
...
+    "@pixi/sprite": "^5.4.0-rc.3",
+    "@pixi/utils": "^5.4.0-rc.3",
     "@rollup/plugin-node-resolve": "^8.4.0",
+    "@types/eventemitter3": "^2.0.2",
     "del": "^2.2.0",
     "jsdoc": "^3.4.0",
     "mkdirp": "^0.5.1",
     "parallelshell": "^2.0.0",
-    "pixi.js": "^5.3.3",
+    "pixi.js": "^5.4.0-rc.3",
```
Comment (bigtimebuddy) [inline package.json:65]: Can this be removed if you're using the leaf packages?

File: package.json [hunk 2]
```diff
@@ -37,29 +37,32 @@
...
-    "@pixi-build-tools/rollup-configurator": "^1.0.3",
-    "@pixi/canvas-renderer": "^5.3.3",
-    "@pixi/constants": "^5.3.3",
-    "@pixi/core": "^5.3.3",
-    "@pixi/display": "^5.3.3",
-    "@pixi/graphics": "^5.3.3",
-    "@pixi/math": "^5.3.3",
-    "@pixi/sprite": "^5.3.3",
+    "@pixi-build-tools/rollup-configurator": "~1.0.11",
+    "@pixi/canvas-renderer": "5.4.0-rc.3",
```
Comment (bigtimebuddy) [inline package.json:51]: Recommend ^6.0.0-X

File: package.json (hunk 1 above)
Comment (SukantPal) [reply package.json]: 👍

File: package.json (hunk 2 above)
Comment (SukantPal) [reply package.json]: Cool, I guess the dependencies can have 5.4.0-X while devDependencies are v6. No breaking changes, _yet_!

Comment (bigtimebuddy) [reply package.json]: Huh? I don't understand, this is kinda confusing. 5.4.0 is not a thing and will never come out. Why not bump everything to 6?

**Comment by SukantPal** [comment]
@bigtimebuddy Removed the dependency on the canvas tilemap plugin & removed registration. That should fix the PIXI reference.

File: package.json (hunk 2 above)
Comment (SukantPal) [reply package.json]: Ok, deps too to v6 :). I though some people might not want to do a major bump but I guess it doesn't matter.

**Pull request merged**